assemble_synApps

```diff
@@ -66,7 +66,7 @@ if ($releasefile eq "default")
 	$modules{'DXPSITORO'} = 'R1-3';
 	$modules{'DEVIOCSTATS'} = '3.1.16';
 	$modules{'ETHERIP'} = 'ether_ip-3-3';
-	$modules{'GALIL'} = 'V3-5';
+	$modules{'GALIL'} = 'V3-6';
```

Comment:
Galil V3-6 does not build on Windows machines, which is why 3-5 is the default. Individual modules can be adjusted for your specifications by using the --set flag.

Reply:
Thanks for pointing that out. I am fine with the option, of course.

Reply:
The Galil driver V3-6 and R3-6-85 have been patched to fix the build issue on Windows.  The Galil repository name has changed recently and tag R3-6-85 was created to align the Galil repo structure to other synApps modules.  I cloned this assemble_synApps repo and altered the assemble_synApps script for the Galil repo changes (see attached).  After the assemble_synApps script completed successfully, I built the software by running "make" in the support folder.  The software built successfully on CentOS stream 9, Windows 10 using visual studio 2019 & 2022.
[assemble_synApps.txt](https://github.com/EPICS-synApps/assemble_synApps/files/14585719/assemble_synApps.txt)
If the Galil driver isn't required, it can be commented out in configure/RELEASE.

Reply:
Thanks a lot for the update on that!